12/umbraco-cms/tutorials/editors-manual/getting-started-with-umbraco/creating-saving-and-publishing-content.md

```diff
@@ -90,7 +90,7 @@ To set up scheduled unpublishing, follow these steps:
 3. Select **Schedule**.
 4.  In the **Scheduled Publishing** window, set the date and time in the **Unpublish at** field.
 
-    ![Schedualed unpublishing.](../../../../../10/umbraco-cms/tutorials/editors-manual/getting-started-with-umbraco/images/Schedule\_Unpublishing\_v9.png)
+    ![Scheduled unpublishing.](../../../../../10/umbraco-cms/tutorials/editors-manual/getting-started-with-umbraco/images/Schedule\_Unpublishing\_v9.png)
```

Comment:
to keep the docs consistent, can you please make this change to v10 and v11 as well?